fast_dp/image_readers.py

```diff
@@ -141,7 +87,17 @@ def XDS_INP_to_dict(inp_text):
             # assume tokens of form key=value key=value
             tokens = useful.replace('=', ' ').split()
             for j in range(useful.count('=')):
-                result[tokens[2*j].strip()] = tokens[2*j+1].strip()
+                key = tokens[2*j].strip()
+                value = tokens[2*j+1].strip()
+                # handle multiple's gracelessly
+                if key in result:
+                    if type(result[key]) is str:
```

Comment:
```suggestion
                    if isinstance(result[key], six.string_types):
```
may need to add ```import six``` to the top of the file

Reply:
In here can ask if type not list => avoid need fgor six